### PR TITLE
Gibson Les Paul Studio

### PR DESCRIPTION
**Pull request by fsapir**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by fsapir** [comment]
https://github.com/instructlab/taxonomy/issues/820

File: knowledge/musical_instruments/les_paul_studio/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/musical_instruments/les_paul_studio/qna.yaml:1]: Please add your GitHub Username here.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please make this change and then we can run the linter.

File: knowledge/musical_instruments/les_paul_studio/qna.yaml (hunk 1 above)
Comment (jjasghar) [reply knowledge/musical_instruments/les_paul_studio/qna.yaml:1]: Please add another commit with that change. Then we can run the linter to check the file(s).

Comment (fsapir) [reply knowledge/musical_instruments/les_paul_studio/qna.yaml:1]: I did this during Summit with on premise support and instruction. Not a coder by trade, could you advise

Comment (jjasghar) [reply knowledge/musical_instruments/les_paul_studio/qna.yaml:1]: Ah ok. 

There are some standard way to add and update PRs, and me walking you through this is unrealistic through comments on the PR. Please google "standard Pull request GitHub" or the like and attempt to update this PR. 

If you don't want to continue that's fully understandable, and please close this PR below so we can remove it from our queue

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the linting issues.

**Comment by jjasghar** [comment]
closed due to combined PR: https://github.com/instructlab/taxonomy/pull/1099

**Pull request closed without merging**